core/amber/src/main/resources/python_udf/topic_modeling_mallet_trainer.py

```diff
@@ -0,0 +1,90 @@
+import logging
+
+# Use gensim 3.8.3
+import gensim
+import gensim.corpora as corpora
+import pandas
+import os
+
+import texera_udf_operator_base
+
```

Comment:
```suggestion
import logging
# to change library's logger setting
logging.getLogger("gensim").setLevel(logging.ERROR)
```
Please add this as the user should be in charge of configuring log levels of the library that the user uses.